include/math.h

```diff
@@ -24,6 +24,12 @@
 #define isinf(x)   __builtin_isinf(x)
 #define signbit(x) __builtin_signbit(x)
 
+#define HUGE_VAL  __builtin_huge_val()
+#define HUGE_VALF (__builtin_huge_valf())
+#define HUGE_VALL (__builtin_huge_vall())
+#define INFINITY  (__builtin_inff())
+#define NAN       (__builtin_nanf(""))
```

Comment:
Why these need (..) around them?

Reply:
I think it was made that way because of how it's defined in `glibc`:
https://github.com/bminor/glibc/blob/master/math/math.h#L59

and `newlib`:
https://github.com/bminor/newlib/blob/master/newlib/libc/include/math.h#L19


IMHO this is a coding convention, not a requirement so we can safely omit `(..)`.

Reply:
Yeah, that was the reason. In fact they aren't needed here, so I removed them.